src/main/java/mate/academy/spring/mapper/Mapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.spring.mapper;
+
+public interface Mapper<I, O, E> {
```

Comment:
```suggestion
public interface Mapper<F, T> {
``` 
better create implementations for response and request separately